Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml

```diff
@@ -10228,12 +10228,12 @@ c.text['SO'].values()[0].each { tex -&gt;&#xD;
         </target>
         <complexParameter name="script">
             <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
-def c = _.context.collector()&#xD;
-&#xD;
-if (dientZurDarstellungVon.p.first()) {&#xD;
```

Comment:
Did you check, if the same condition is used somewhere else in the alignment? If not, I would recommend to do so in order to make sure that no one is running into it with the next test case.